R/help.R

```diff
@@ -238,33 +248,63 @@ help_formals_handler.python.builtin.object <- function(topic, source) {
   NULL
 }
 
+sphinx_doc_params_pattern <- function() { "\nParameters\n+[-]+" }
```

Comment:
We can probably just use the regular expression directly (no need for a separate function since it's not called anywhere else)